<launch>
  <!--- Run AMCL -->
  <!--- We load ACML here with diff=true to support our differential drive robot -->
  <include file="$(find amcl)/examples/amcl_diff.launch" />

  <!-- Run the map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find auto_control)/map/map.yaml"/>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">


    <rosparam file="$(find autu_control)/launch/trajectories/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find autu_control)/launch/trajectories/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find autu_control)/launch/trajectories/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find autu_control)/launch/trajectories/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find autu_control)/launch/trajectories/config/base_local_planner_params.yaml" command="load"/>


    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <param name="controller_frequency" value="10.0" />
  </node>
</launch>